### PR TITLE
Text Aligned to the Start & dependencies updated #69

### LINKED ISSUE #69: Text alignment in Forum Edit text
Currently the edit text in forumn aligns text centrally, but it doesn't look, so the text inside the edit text should start from left.
The file to be changed is activity_forum.xml.

### PR DESCRIPTION
**Pull request by Pranav915**
Fixes #69 
## Description

Fix #69

1) Text inside EditText in Forumn is aligned to start.
2) New Versions of Dependencies are added in gradle files.
## Checklist:

- [x] I used the Reformat code option in android studio for the files I changed
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] The App is running Fine and as Expected In my Emulator or Physical Device
- [x] I have checked my code and corrected any misspellings
- [x] My changes didn't cause any issues for the other functionalities present before.
- [x] This PR is created From a branch which is upto date with the master branch

### CODE REVIEW

File: app/src/main/res/layout/activity_forumn.xml [hunk 1]
```diff
@@ -23,24 +23,20 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/chat_recycler"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_marginTop="40dp"
-        android:layout_marginBottom="40dp"
-        app:layout_constraintBottom_toTopOf="@id/message_input"
+        android:layout_width="414dp"
+        android:layout_height="610dp"
```
Comment (Amar-2003) [inline app/src/main/res/layout/activity_forumn.xml:27]: @Pranav915  pls revert these layout width and height changes

File: app/src/main/res/layout/activity_forumn.xml [hunk 2]
```diff
@@ -23,24 +23,20 @@
...
+        android:layout_marginTop="10dp"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/forum_head" />
 
     <com.google.android.material.textfield.TextInputEditText
         android:id="@+id/message_input"
-        android:layout_width="match_parent"
+        android:layout_width="370dp"
```
Comment (Amar-2003) [inline app/src/main/res/layout/activity_forumn.xml:34]: Revert this width changed

**Review by Amar-2003 — CHANGES_REQUESTED** [review]
@Pranav915 left a few imp changes

File: app/src/main/res/layout/activity_forumn.xml (hunk 1 above)
Comment (Pranav915) [reply app/src/main/res/layout/activity_forumn.xml:27]: Ok

**Comment by Amar-2003** [comment]
@Pranav915 , there is also a merge conflict please look into it

**Comment by Amar-2003** [comment]
@Pranav915 why did you close this ??

**Pull request closed without merging**